plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -204,6 +204,15 @@ public void testSystemTable()
                 .contains("orders");
     }
 
+    @Test
+    public void testPartitionedTables()
+            throws Exception
+    {
+        execute("CREATE TABLE test_part_tbl (id int not null, payload varchar, logdate  date not null) PARTITION BY RANGE (logdate)");
+        assertThat(computeActual("SHOW TABLES").getOnlyColumnAsSet())
+                .contains("test_part_tbl");
```

Comment:
Please use `TestTable`. Also, it would be nice to test `SELECT` query as well.

Reply:
The actual partition tables should be created before data inserts, so `TestTable` can't be used with rows.
Meanwhile, I added the `SELECT` tests with explicit `CREATE` and `INSERT` statements

Reply:
`TestTable` allows creating tables without rows.

Reply:
Ye, it's much cleaner